src/main/java/cinema/model/dto/moviesession/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,33 @@
+package cinema.model.dto.moviesession;
+
+import java.time.LocalDateTime;
+
+public class MovieSessionRequestDto {
+    private Long movieId;
+    private Long cinemaHallId;
+    private LocalDateTime showTime;
```

Comment:
```suggestion
    private String showTime;
```
LocalDateTime doesn't work with json

Reply:
Thanks